src/DotVVM.Core/ComponentModel/DataAnnotations/ClientFormatAttribute.cs

```diff
@@ -9,15 +9,24 @@ namespace System.ComponentModel.DataAnnotations
     /// Checks whether the value matches the format specified by DotVVM control (e.g. TextBox).
     /// The value can be empty string or null. If it is not, it must match the FormatString that the control specifies.
     /// </summary>
+    [Obsolete("The client format is enforced by default. If you need to disable it use ClientFormatAttribute", true)]
```

Comment:
I'd also include a note that `ClientFormatAttribute` also allows you to configure the behavior.